src/EmailAddress.php

```diff
@@ -547,6 +547,9 @@ protected function idnToAscii($email)
      */
     protected function idnToUtf8($email)
     {
+        if (strlen($email) == 0)
```

Comment:
PSR-2 missing opening and closing parentheses: http://www.php-fig.org/psr/psr-2/#5-1-if-elseif-else